helm/ck-application/files/default_conf.yaml

```
@@ -0,0 +1,14 @@
+server {
```

yamllint

[error] wrong new line character: expected \n

(new-lines)